### PR TITLE
Integrate Travis CI checks into the repo

### PR DESCRIPTION
**Pull request by serhii73**
Added config for Travis CI and added a badge to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
+[![Build Status](https://travis-ci.com/kpi-web-guild/django-girls-blog-serhii73.svg?branch=master)](https://travis-ci.com/kpi-web-guild/django-girls-blog-serhii73)
 # django-girls-blog-serhii73
-##### This is a simple blog based on the djangogirl's tutorial.
\ No newline at end of file
+##### This is a simple blog based on the djangogirl's tutorial.
```
Comment (webknjaz) [inline README.md]: Why did you change this?

Comment (serhii73) [reply README.md]: I solved the conflict in the browser and probably I pressed the key.
In the browser, it's not very convenient to do. My fault.

Comment (webknjaz) [reply README.md]: You can as well do this using Git CLI.

**Comment by serhii73** [comment]
@webknjaz I think we can to merge.
